File: lib/features/restaurant/domain/entities/restaurant.dart
class Restaurant {
  final String id;
  final String name;
  final String category;
  final String imageUrl;
  final String area;
  final double rating;
  final String time;

  const Restaurant({
    required this.id,
    required this.name,
    required this.category,
    required this.imageUrl,
    required this.area,
    this.rating = 4.5,
    this.time = '25 menit',
  });
}

File: backend/internal/models/restaurant.go
package models

type Restaurant struct {
	ID       string  `json:"id"`
	Name     string  `json:"name"`
	Category string  `json:"category"`
	ImageURL string  `json:"image_url"`
	Area     string  `json:"area"`
	Rating   float64 `json:"rating"`
	Time     string  `json:"time"`
}

type RestaurantResponse struct {
	Success bool         `json:"success"`
	Message string       `json:"message"`
	Data    []Restaurant `json:"data"`
}